reposcan/pkgtree.py

```diff
@@ -89,6 +89,15 @@ def dump(self):
             LOGGER.info("Removing old dump %s", fname)
             os.unlink(fname)
 
+        self._update_pkgtree_timestamp()
+
+    def _update_pkgtree_timestamp(self):
```

Comment:
Could you please insert here timestamp of the pkgtree (defined on line 58) instead of current timestamp, so the values returned by the /dbchange API and in /pkgtree output match?